apps/web-giddh/src/app/services/http-wrapper.service.ts

```diff
@@ -75,7 +75,7 @@ export class HttpWrapperService {
         options.headers["Session-Id"] = this.generalService.sessionId;
         options.headers["Content-Type"] = "application/json";
         options.headers["Accept"] = "application/json";
-        options.headers["X-Tenant"] = this.generalService.getUtmParameter("X-Tenant");
+        // options.headers["X-Tenant"] = this.generalService.getUtmParameter("X-Tenant");
```

Comment:
Informed to Leena for handle this if this on Api if task is not going to prod.

Reply:
Done